frontend-ssr/components/cli_git/GitCli_CC.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { uuid } from '../../services/Utils';
+
+import styles from './GitCli.module.scss';
+
+interface Props {
+  commands: RegExp[] | string[],
+  successMessages: string[],
+}
+
+interface State {
+  currentIndex: number,
+  messages: Message[],
+  btnMessage: string,
+  loading: boolean,
+}
+
+interface Message {
+  text: string,
+  type: MessageType,
+  time: number,
+  id: string,
+}
+
+enum MessageType {
+  COMMAND = 'command',
+  ERROR = 'error',
+  SUCCESS = 'success'
+}
+
+class matchCommandMessageClassComponent extends React.Component<Props, State, Message> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentIndex: 0,
+      messages: [],
+      btnMessage: 'Next',
```

Comment:
Here we should treat the case in which we have just one item in the Array. Meaning it should be like this:

`btnMessage: props.commands.length === 1 ? 'Done' : 'Next'`